content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -172,20 +170,167 @@ Use `kubectl delete` to remove the Pod when you're finished:
+behaving as you expect and you'd like to add additional troubleshooting
+utilities to the Pod.
+
+For example, maybe your application's container images is built on busybox
+but you need debugging utilities not included in busybox. You can simulate
+this scenario using `kubectl run`:
+
+```shell
+kubectl run myapp --image=busybox --restart=Never -- sleep 1d
+```
+
+Run this command to create a copy of `myapp` named `myapp-copy` that adds a
+new ubuntu container for debugging.
+
+```shell
+kubectl debug myapp -it --image=ubuntu --share-processes --copy-to=myapp-debug
+```
+
+```
+Defaulting debug container name to debugger-w7xmf.
+If you don't see a command prompt, try pressing enter.
+root@myapp-debug:/#
+```
+
+There are a few things worth noting in this invocation of `kubectl debug`:
+
+* `kubectl debug` automatically generates a container name  if you don't choose
+  one using the `--container` flag.
+* The `-i` flag causes `kubectl debug` to attach to the new container by
+  default.  You can prevent this by specifying `--attach=false`. If your session
+  becomes disconnected you can reattach using `kubectl attach`.
+* The `--share-processes` causes the Pod to use one process namespace so that
+  you'll be able to see other processes from other containers using `ps`.
+
+Don't forget to clean up the debugging Pod when you're finished with it:
+
+```shell
+kubectl delete pod myapp myapp-debug
+```
+
+### Copying a Pod while changing its command
+
+Sometimes it's useful to change the command for a container, for example to
+add a debugging flag or because the application is crashing.
+
+To simulate a crashing application, use `kubectl run` to create a container
+that immediately exists:
+
+```
+kubectl run --image=busybox myapp -- false
+```
+
+You can see using `kubectl describe pod myapp` that this container is crashing:
+
+```
+Containers:
+  myapp:
+    Image:         busybox
+    ...
+    Args:
+      false
+    State:          Waiting
+      Reason:       CrashLoopBackOff
+    Last State:     Terminated
+      Reason:       Error
+      Exit Code:    1
+```
+
+You can use `kubectl debug` to create a copy of this Pod with the command
+changed to an interactive shell:
+
+```
+kubectl debug myapp -it --copy-to=myapp-debug --container=myapp -- sh
+```
+
+```
+If you don't see a command prompt, try pressing enter.
+/ #
+```
+
+Now you have an interactive shell that you can use to perform tasks like
+checking filesystem paths or running the container command manually. There
+are a few things worth noting about this invocation of `kubectl debug`:
+
+* Since you want to change the command of a specific container you must
+  specify its name using `--container` or `kubectl debug` will instead
+  create a new container to run the command you specified.
+* The `-i` flag causes `kubectl debug` to attach to the container by default.  You can prevent this by specifying `--attach=false`. If your session becomes
```

Comment:
Line 271, Could you add a carriage return before, "You can prevent ..."

Reply:
Wow. Yes. Not sure how that happened.